README.md

```diff
@@ -17,21 +17,21 @@ This tool is primarily for the developers on the Fractally team, or other develo
 1. Clone this psibase-contributor repo to your development PC.
 2. Open this repo in vscode
 3. Execute the VSCode command to relaunch this repository inside a Docker container <br/> ![Reopen in Container](/img/build-command.png)
-4. Run a full build by clicking the corresponding button in the blue status bar at the bottom of the window in VSCode <br/> ![Execute Build](/img/full-build.png)
+4. Run a full build by clicking the corresponding button in the status bar at the bottom of the window in VSCode <br/> ![Execute Build](/img/full-build.png)
 
 ## Note on HTTPS
 
-Running psinode using a database with the name "psinode_db" within the container will use the default psinode config file located at `psinode/configs/dev.config`. This is configured to automatically run psinode locally over https. Http is exposed within the container at port 8079, but the 8080 exposed outside the container is https.
+Running psinode using a database with the name "psinode_db" within the container will use the default psinode config file located in your dev environment at `$PSINODE_PATH/psinode_db/http.config`. This is configured to automatically run psinode locally over https. Http is exposed within the container at port 8079, but the 8080 exposed outside the container is https.
```

Comment:
This seems a bit mixed.
- There is a file called `http.config`, in the `image-builders` repo, which could be called the default configuration file, but it is not available by that name in the dev environment.
- There is a file called `$PSINODE_PATH/psinode_db/config`. I would not call this a default config file, as it's the actual config file that psinode loads.

Reply:
Fixed